subprojects/snapshots/src/test/groovy/org/gradle/internal/fingerprint/impl/PathNormalizationStrategyTest.groovy

```diff
@@ -92,7 +93,7 @@ class PathNormalizationStrategyTest extends Specification {
     }
 
     def "sensitivity NAME_ONLY"() {
```

Comment:
Should this test class include tests for the include/exclude empty directories cases?

Reply:
Yeah probably.  Done.